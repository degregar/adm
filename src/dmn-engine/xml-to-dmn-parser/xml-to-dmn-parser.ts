export class XmlToDmnParser {
  public parse(xml: string) {}
}
