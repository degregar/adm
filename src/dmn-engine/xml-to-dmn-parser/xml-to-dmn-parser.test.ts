import * as fs from "fs"
import * as path from "path"

import { XmlToDmnParser } from "./xml-to-dmn-parser"

it("should parse a dmn xml file", () => {
  // given
  const parser = new XmlToDmnParser()
  const xml = fs.readFileSync(path.join(__dirname, "__tests__", "single-hit-unique.xml"), "utf8")

  // when
  const dmn = parser.parse(xml)

  // then
  expect(dmn).toMatchSnapshot()
})
