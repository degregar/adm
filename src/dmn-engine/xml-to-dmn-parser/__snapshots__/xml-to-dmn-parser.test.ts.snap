// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should parse a dmn xml file 1`] = ``;
